client/flutter/android/fastlane/README.md

```diff
@@ -0,0 +1,39 @@
+fastlane documentation
+================
+# Installation
+
+Make sure you have the latest version of the Xcode command line tools installed:
+
+```
+xcode-select --install
```

Comment:
Do we need xcode for android?

Reply:
No, you're right. This is a generated file. This is definitly an issue with fastlane. These README files could be removed.